ddtrace/_trace/tracer.py

```diff
@@ -59,7 +59,7 @@
 from ddtrace.internal.utils.formats import format_trace_id
 from ddtrace.internal.writer import AgentWriterInterface
 from ddtrace.internal.writer import HTTPWriter
-from ddtrace.version import get_version
+from ddtrace.version import __version__
```

Comment:
This is strictly a style thing, but I think using the public alias `ddtrace.__version__` here is a little more aligned with the philosophy of dogfooding.